cou/exceptions.py

```diff
@@ -147,3 +147,22 @@ def __init__(self, message: str, exit_code: int) -> None:
 
 class ApplicationNotSupported(COUException):
     """COU exception when the application is known but not supported by COU."""
+
+
+class SnapVaultNotInstalled(COUException):
```

Comment:
We should not limit the user to use snap

Reply:
It only fire during the vault upgrading, and we use snap vault to unseal. So it's required.